tests/pages/dashboards-index.page.ts

```diff
@@ -158,6 +162,10 @@ export class DashboardsIndexPage {
     this.dashboardFilter = page.getByRole('form', {
       name: 'Filter dashboards',
     });
+    //Added View Button
```

Comment:
If you run `yarn lint:fix` it should fix a lot of spacing issues like adding a space after the comment `//`.